src/objects.js

```diff
@@ -1,41 +1,68 @@
 const createPerson = (name, age) => {
   // your code here
+  const person = new Object();
+  person.name = name, 
+  person.age = age;
+  
+  return person;
 };
 
 const getName = object => {
   // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
   // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
   // your code here
+  if (object[property]) {
+    return true;
+  }
+  return false;
```

Comment:
The logic works, but it can be made a bit shorter.

When you are looking for return a boolean as you are here, we could use a ternary:
```
  return object[property] ? true : false;
```
While this works, it could be reduced further by using a double negation:
```
  return !!object[property];
```
Much cleaner :)